controller/tests/data/standard/output/lab-form.html

```diff
@@ -34,8 +34,6 @@
       <option value="lighthouse.ceres/library/sketchbook:recommended@sha256:5678">Recommended (Weekly 2077_43)</option>
       <option value="lighthouse.ceres/library/sketchbook:w_2077_43@sha256:5678">Weekly 2077_43</option>
       <option value="lighthouse.ceres/library/sketchbook:d_2077_10_23@sha256:1234">Daily 2077_10_23</option>
-      <option value="lighthouse.ceres/library/sketchbook:latest_weekly">latest_weekly</option>
-      <option value="lighthouse.ceres/library/sketchbook:latest_daily">latest_daily</option>
```

Comment:
Oh, this is the reason why a policy of not including unknown tags doesn't work. If we use a Docker source, all alias tags are unknown tags unless they correspond to a pre-pulled image. I'm not sure how I feel about that. I think it means we need to include all unknown tags for as long as we support Docker sources.

Reply:
latest_weekly and latest_daily are currently alias tags that only appear in the dropdown and are not displayed on the main menu. Currently the underlying images themselves are always prepulled given our current prepull policy, so even with Docker the tags would be resolved to alias tags, but this would not be the case if, say, the menu was configured to hide dailies. But I think in that case we wouldn't want to make latest_daily disappear from the dropdown, would we?

Reply:
....I...don't know?  I guess I'll make unknown, like alias, return `None` and therefore show all the unknown tags, but yuck, and that means I need to clean up Ross's test tag in GAR.

I really don't like showing all of the broken tags we will have accumulated over the years just to be able to show latest_daily/latest_weekly in the (presumably rare) cases we don't prepull them.

Reply:
...I guess we could just put an explicit policy on Unknown again, so I can set it to `number: 0` for GAR...

Reply:
I'm leaving it at `None` to match existing behavior, but...I'm ag'in' it.